app/src/main/java/com/usepace/android/messagingcenter/screens/sendbird/SendBirdChatAdapter.java

```diff
@@ -940,6 +962,64 @@ public void onClick(View v) {
         }
     }
 
+    private class MyAudioMessageHolder extends RecyclerView.ViewHolder {
```

Comment:
normally  I prefer to declare it as static class. non static works as well. :)

Reply:
Actually I'm following with the holder standards in the same class, we have 8 holders both are using the same way

Reply:
that is fine, heap wise static is more good but no problem let us keep it non static. will see later if needed.